tests/monotouch-test/HealthKit/CdaDocumentSampleTest.cs

```diff
@@ -57,11 +57,16 @@ public void Error ()
 #else
 					var ex = Assert.Throws<MonoTouchException> (action, "Exception");
 #endif
+
+#if MONOMAC
+					Assert.That (ex.Message, Does.Match ("startDate.*and endDate.*exceed the maximum allowed duration for this sample type"), "Exception Message");
```

Comment:
We don't get this on Mac Catalyst too?

Reply:
Good question, let me re-run them locally

Reply:
@rolfbjarne Confirmed that Maccatalyst does pass as it is.